release-notes/CREDITS-2.x

```diff
@@ -104,6 +104,6 @@ Samantha Williamson (samwill@github)
   * Contributed fix to #148: Allow strict `LocalDate` parsing
    (2.11.0)
 
-Moritz Orth (morth@github)
+Moritz Orth
```

Comment:
Actually the e-mail at issue was my e-mail, and GitHub was blocking me from pushing the changes to this file until unchecked a setting.

Reply:
fixed